Libraries/CustomComponents/NavigationExperimental/NavigationCard.js

```diff
@@ -87,6 +87,8 @@ class NavigationCard extends React.Component<any, Props, any> {
       ...props,
     } = this.props;
 
+    const defaultStyle = NavigationCardStackStyleInterpolator.forHorizontal(props);
```

Comment:
This is unnecessary.
We should let people to overwrite this animation with custom style.

In your example, you could just do this:

```
_renderCard(/*NavigationSceneRendererProps*/ props) {
  return (
    <NavigationCard
      {...props}
      style={[
        NavigationCardStackStyleInterpolator.forHorizontal(props),
        styles.card,
      ]}
    />
  );
}
```